### PR TITLE
Add support to retrieve time record-types as values using queryRow()

### PR DESCRIPTION
**Pull request by kaneeldias**
## Purpose
Adds support to retrieve the time record types from `queryRow()` method as values.
- `Time:Civil`
- `Time:Date`
- `Time:TimeOfDay`

Fixes: https://github.com/ballerina-platform/ballerina-standard-library/issues/1939

## Examples
```
ParameterizedQuery sqlQuery = `SELECT datetime_type FROM DateTimeTypes WHERE row_id = 1`;
time:Civil retrievedValue = check dbClient->queryRow(sqlQuery);

time:Civil timestampTypeRecord = {year: 2017, month: 2, day: 3, hour: 11, minute: 53, second: 0};
test:assertEquals(retrievedValue, timestampTypeRecord);
```

## Checklist
- [x] Linked to an issue
- [x] Updated the changelog
- [x] Added tests

### CODE REVIEW

File: native/src/main/java/io/ballerina/stdlib/sql/utils/Utils.java [hunk 1]
```diff
@@ -1178,4 +1178,20 @@ public static String getBTypeName(Type ballerinaType) {
         }
         return ballerinaType.getName();
     }
+
+    public static boolean isKnownRecordType(Type ballerinaType) {
+        String[] knownRecordTypes = {
```
Comment (niveathika) [inline native/src/main/java/io/ballerina/stdlib/sql/utils/Utils.java:1183]: Let's make this a util class constant. Do we only support civil and utc? I thought TimeOfDay and others are also supported.

File: native/src/main/java/io/ballerina/stdlib/sql/utils/Utils.java [hunk 2]
```diff
@@ -1178,4 +1178,20 @@ public static String getBTypeName(Type ballerinaType) {
...
+                Constants.SqlTypes.CIVIL,
+                Constants.SqlTypes.DATE_RECORD,
+                Constants.SqlTypes.TIME_RECORD
+        };
+        String typeName = getBTypeName(ballerinaType);
+        for (String type: knownRecordTypes) {
```
Comment (niveathika) [inline native/src/main/java/io/ballerina/stdlib/sql/utils/Utils.java:1189]: We can use contains method on a list. Much cleaner

File: native/src/main/java/io/ballerina/stdlib/sql/utils/Utils.java (hunk 1 above)
Comment (kaneeldias) [reply native/src/main/java/io/ballerina/stdlib/sql/utils/Utils.java]: Made a class constant in 2bf215b

Civil, Date and TimeOfDay are supported

File: native/src/main/java/io/ballerina/stdlib/sql/utils/Utils.java (hunk 2 above)
Comment (kaneeldias) [reply native/src/main/java/io/ballerina/stdlib/sql/utils/Utils.java]: Fixed in 2bf215b

File: native/src/main/java/io/ballerina/stdlib/sql/utils/Utils.java [hunk 3]
```diff
@@ -1178,4 +1181,9 @@ public static String getBTypeName(Type ballerinaType) {
         }
         return ballerinaType.getName();
     }
+
+    public static boolean isKnownRecordType(Type ballerinaType) {
```
Comment (niveathika) [inline native/src/main/java/io/ballerina/stdlib/sql/utils/Utils.java:1185]:
```suggestion
    public static boolean isSupportedRecordType(Type ballerinaType) {
```

**Review by niveathika — APPROVED** [review]

**Pull request merged**